pbrEngine/engine.py

```diff
@@ -157,11 +155,26 @@ def _crash(reason=None):
               CAUTION: broken- needs fixing or removal
         arg3: <movename> name of the move used.
               CAUTION: The attacking pkmn might not have this attack (e.g. Metronome)
+        arg4: <success> whether the move is successful.
+              I.e., it doesn't miss or fail.  For a move with multiple targets, this is 
+              true if the move succeeds against at least one target.
         arg4: <obj> object originally returned by the action-callback that led
```

Comment:
arg5, and this documentation was missing "teams" before too.

Reply:
I need to update documentation on a few of the callbacks, do you want those changes in this PR or a new one?

Reply:
however you prefer it, I don't mind either

Reply:
I will make them in a separate PR this weekend